demisto_sdk/commands/content_graph/objects/modeling_rule.py

```diff
@@ -53,3 +54,19 @@ def match(_dict: dict, path: Path) -> bool:
             ):
                 return True
         return False
+
+    def get_related_content(self) -> Dict[RelatedFileType, dict]:
+        related_content_ls = super().get_related_content()
+        related_content_ls.update(
+            {
+                RelatedFileType.SCHEMA: {
+                    "path": [str(self.path).replace(".yml", "_Schema.json")],
```

Comment:
why not to keep them as `Path` instead of `str`?

Reply:
easier to handle when calling path in the rest of the code.